src/components/ContentList/ContentList.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { arrayOf, object } from 'prop-types';
+import './contentList.scss';
+
+const ContentList = ({ contents }) => (
+  contents.map((content) => {
+    return (
+      <div key={content.id} className="content-item">
+        <a href={content.link}>
+          <p>{content.name}</p>
+        </a>
+      </div>
+    );
+  })
+);
```

Comment:
Why can't we structure this to be more concise and readable with arrow function return? Like this 👇 

```js
const ContentList = ({ contents }) => contents.map((content) => (
 <div key={content.id} className="content-item">
   <a href={content.link}>
      <p>{content.name}</p>
   </a>
 </div>)
);
```